install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -301,10 +362,23 @@ metadata:
 spec:
   value: connection.sent.bytes | 0
   dimensions:
-    source_service: source.service | "unknown"
+    source_name: source.name | "unknown"
+    source_namespace: source.namespace | "unknown"
+    source_workload: source.workload.name | "unknown"
+    source_workload_namespace: source.workload.namespace | "unknown"
+    source_principal: source.principal | "unknown"
+    source_app: source.labels["app"] | "unknown"
     source_version: source.labels["version"] | "unknown"
-    destination_service: destination.service | "unknown"
+    destination_name: destination.name | "unknown"
+    destination_namespace: destination.namespace | "unknown"
+    destination_workload: destination.workload.name | "unknown"
+    destination_workload_namespace: destination.workload.namespace | "unknown"
+    destination_principal: destination.principal | "unknown"
+    destination_app: destination.labels["app"] | "unknown"
     destination_version: destination.labels["version"] | "unknown"
+    destination_service: destination.service.name | "unknown"
+    destination_service_name: destination.service.name | "unknown"
+    destination_service_namespace: destination.service.namespace | "unknown"
```

Comment:
I'm not sure if we want all of these namespace dimensions. In kubernetes, at least, they are all the same. Doesn't hurt though...

Reply:
Namespace would be good to filter on. People probably want to filter out all the traffic to/from istio-system when looking at their app.

Reply:
Agree, but we don't need three labels that are always equal.

Reply:
@kyessenov in k8s they are the same. i'm not sure that will be the general case necessarily however. I don't have strong feelings here, expect that it seemed somewhat more natural to include the three concepts.

I could see dropping `(source|destination).workload_namespace` at least.

Thoughts?